### PR TITLE
Include types in build output

### LINKED ISSUE #51: Include types in build output
When using `rLogin` in a typescript project, it asks for the missing types.
Currently it is solved by creating a `.d.ts` file with `declare module '@rsksmart/rlogin'`

### PR DESCRIPTION
**Pull request by diegograsso**
Fix the types export
Closes #51

### CODE REVIEW

**Review by jessgusclark — APPROVED** [review]
This worked for me when tested with the RIF Id Manager. I'll wait for @ilan to take a look before we merge it.

File: sample/front/index.html
Comment (ilanolkies) [inline sample/front/index.html:42]: Seems like this a breaking change. Can we still use `main.js` or do we need `index.js` for any reason?

Comment (ilanolkies) [reply sample/front/index.html:42]: Cannot be changed

**Review by ilanolkies — APPROVED** [review]

**Pull request merged**
